### PR TITLE
DM-32767: Add basic INFO level logging

### PR DESCRIPTION
**Pull request by brendal4**
{Summary of changes. Prefix PR title with ticket handle.}

****

- [ ] Passes Jenkins CI.
- [ ] Documentation is up-to-date.

Preview the docs at: https://pipelines.lsst.io/v/DM-FIXME

### CODE REVIEW

File: python/lsst/verify/bin/dispatchverify.py [hunk 1]
```diff
@@ -203,6 +204,7 @@ def build_argparser():
 def main():
     """Entrypoint for the ``dispatch_verify.py`` command line executable.
     """
+    logging.basicConfig(level=logging.INFO, stream=sys.stdout)
```
Comment (timj) [inline python/lsst/verify/bin/dispatchverify.py:207]: You might want to consider playing with the log format as well by adding something like:

```python
format="{name} {levelname}: {message}", style="{")
```
to the end of the `basicConfig` call.

**Comment by SimonKrughoff** [comment]
For future reference, the PR name should be prefixed with the ticket number like "DM-32767: Add basic INFO level logging"

File: python/lsst/verify/bin/dispatchverify.py (hunk 1 above)
Comment (SimonKrughoff) [reply python/lsst/verify/bin/dispatchverify.py:207]: I think Tim's suggestion is a good one, but you are the one who will have to read the outputs, so I leave it up to you.

**Review by SimonKrughoff — APPROVED** [review]

**Comment by timj** [comment]
Also if this is your first time doing a rebase, don't hit that Update Branch button on this page. You need to do the:

```
git checkout master
git pull
git checkout tickets/DM-32767
git rebase main
git push -f
```
on your local computer.

**Comment by brendal4** [comment]
OK, thanks for the suggestions. I'm going to try adding this line that Tim suggests, run it through Jenkins, do a new PR, then rebase my branch. Thanks, all!

**Comment by timj** [comment]
@brendal4 why did you close this pull request? You can force push to the same branch and everything works. It also means that you retain the history of the discussion on the pull request. Given that you have to reuse the branch name you will have to reopen the pull request anyhow. I will reopen it so that we know it's still active.

**Comment by brendal4** [comment]
Whoops, I thought that I had to make a new pull request each time there were new changes. Now I know. Thanks for catching that!

**Comment by SimonKrughoff** [comment]
Looks good to me

**Pull request merged**